### ISSUE TITLE
Support username and password tokens from dockercfg auths

### ISSUE THREAD

**Issue by tpokki** [GoogleContainerTools/jib#3364]
**Environment**:

- *Jib version:* 3.1.2 
- *Build tool:* maven 3.6.3
- *OS:* macOS

**Description of the issue**:
Jib is missing support for `username` and `password` fields in dockercfg auths block.
Example
```
{
  "auths": {
      "registry.example.com": {
           "username": "someuser",
           "password": "somepw"
       }
}
```

**Expected behavior**:
Jib to pick up username and password fields from dockercfg (and not just `credHelper` and `auth`). 

**Steps to reproduce**:

  1. Add username and password for your registry into your config.json as described above
  2. Build
  3. Jib does not pick up username and password

**Additional Information**: 

I have implemented the necessary changes to add support in here:
https://github.com/tpokki/jib/commit/479a5dfcebfe1d451bd43d00b7e7adddbc16080f

If you approve, I'll create PR. 

Thanks,
Toni

**Comment by chanseokoh** [comment]
I think this is great. Create a PR (with a test), and we'll review it. Thanks a lot!

**Comment by chanseokoh** [comment]
Fixed by #3365.

**Comment by chanseokoh** [comment]
@tpokki we released Jib plugins 3.1.4 with the fix.

**Comment by chanseokoh** [comment]
Thanks for your contribution!
